Makefile

```diff
@@ -142,15 +142,10 @@ dist: gen-doc
 	go run tools/create-artifacts/main.go -version ${VERSION} -commit ${GIT_COMMIT} -goversion ${GO_VERSION}
 
 BEHAT_BIN=vendor-bin/behat/vendor/bin/behat
-# behat config file for core api tests
-CORE_BEHAT_YML=$(PATH_TO_APITESTS)/tests/acceptance/config/behat-core.yml
 
 test-acceptance-api: vendor-bin/behat/vendor
 	BEHAT_BIN=$(BEHAT_BIN) $(PATH_TO_APITESTS)/tests/acceptance/run.sh --type api
```

Comment:
This `test-acceptance-api` make target is used to run the local tests (apiVirtualViews) in reva, the test only runs for the `master` branch so there is no issue with the `edge` branch in removing that.